src/native/tools/ssl_napi.cpp

```diff
@@ -87,7 +89,10 @@ class Hasher : public Napi::ObjectWrap<HASHER_ARGS>
     {
         Napi::Env env = info.Env();
         submit_and_flush(0, 0, HASH_LAST);
-        return Napi::Buffer<uint8_t>::Copy(env, reinterpret_cast<const uint8_t*>(&hash_ctx_digest(&_ctx)[0]), LEN);
+        uint32_t digest[NWORDS];
+        for (unsigned long i = 0; i < NWORDS; i++)
+            digest[i] = WORDS_BE ? be32toh(hash_ctx_digest(&_ctx)[i]) : le32toh(hash_ctx_digest(&_ctx)[i]);
+        return Napi::Buffer<uint8_t>::Copy(env, reinterpret_cast<const uint8_t*>(digest), NWORDS * 4);
```

Comment:
I think we should be able to just return like this (didn't test) -
```suggestion
        return Napi::Buffer<uint32_t>::Copy(env, digest, NWORDS);
```

Reply:
both compilation and unit test passed - so changed to this